apps/ui/src/components/views/setup-view/hooks/use-cli-status.ts

```
@@ -9,6 +9,9 @@ interface UseCliStatusOptions {
 }
 const logger = createLogger('CliStatus');
 
+// Create logger once outside the hook to prevent infinite re-renders
+const logger = createLogger('CliStatus');
```

Biome (2.1.2)

[error] Shouldn't redeclare 'logger'. Consider to delete it or rename it.

'logger' is defined here:


(lint/suspicious/noRedeclare)